src/libraries/System.Linq.Queryable/src/System/Linq/EnumerableQuery.cs

```diff
@@ -110,9 +110,9 @@ private IEnumerator<T> GetEnumerator()
             return _enumerable.GetEnumerator();
         }
 
-        public override string ToString()
+        public override string? ToString()
         {
-            ConstantExpression c = _expression as ConstantExpression;
+            ConstantExpression? c = _expression as ConstantExpression;
             if (c != null && c.Value == this)
```

Comment:
We normally update such `as` expression to `is` 
`if ( _expression is ConstantExpression c && c.Value == this)`

Reply:
Sure, I'm wondering though whether it's appropriate for the scope of annotation. Wouldn't we generally want to avoid changes that could potentially result in slightly different IL?

Reply:
This is one kind of comments I have had before in the scope of annotations PR. Anyways by my understanding  our direction is not to increase IL size, and I assume it wouldn't increase IL size.

Personally I don't like to do any other refactoring with annotations though.